README.md

```diff
@@ -152,6 +152,7 @@ brew upgrade glab
 Make sure you have snap installed on your Linux Distro (https://snapcraft.io/docs/installing-snapd).
 1. `sudo snap install --edge glab`
 1. `sudo snap connect glab:ssh-keys` to grant ssh access
+1. `sudo snap connect glab:config-git` to grant read access to `/etc/gitconfig`
```

Comment:
I would that we add a reference to the issue that led to this PR so users know why that access should be granted
```suggestion
1. `sudo snap connect glab:config-git` to grant read access to `/etc/gitconfig`. See https://github.com/profclems/glab/issues/597
```

Reply:
Sounds good, thanks!